pkg/products/threescale/reconciler.go

```diff
@@ -577,8 +585,9 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient k8scl
 	resourceRequirements := r.installation.Spec.Type != string(integreatlyv1alpha1.InstallationTypeWorkshop)
 	apim := &threescalev1.APIManager{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      apiManagerName,
-			Namespace: r.Config.GetNamespace(),
+			Name:        apiManagerName,
+			Namespace:   r.Config.GetNamespace(),
+			Annotations: map[string]string{"apps.3scale.net/disable-apicast-service-reconciler": "true"},
```

Comment:
I see that we are adding annotations here and also in a sperate function. Do we need both? If we are using it here I would suggest using the flow below. check if the annotations are nil. If it's nil add the string map. the above will overwrite existing annotations that are added which we don't want to do in case a user wants to add any to the list.

Reply:
Ah okay I didn't realise they overwrite existing ones. pushing this and the change below now